src/main/java/com/revature/services/JMeterServices.java

```diff
@@ -0,0 +1,248 @@
+            String basePath = input.getBasePath();
+            Map<String, Path> endpoints = input.getPaths();
+
+            // each path
+            for (String path : endpoints.keySet()) {
+                Path pathOperations = endpoints.get(path);
+                Map<HttpMethod, Operation> verbs = pathOperations.getOperationMap();
+
+                // each verb/operation
+                for (HttpMethod verb : verbs.keySet()) {
+                    HTTPSampler element = new HTTPSampler();
+
+                    // domain
+                    element.setDomain(splitHost[0]);
+                    try {
+                        // port
+                        element.setPort(Integer.parseInt(splitHost[1]));
+                    
+                    // couldn't parse port
+                    } catch (NumberFormatException e) {
+                        return null;
+                    } catch (IndexOutOfBoundsException e) {
+                        return null;
+                    }
+
+                    // path
+                    if (basePath.equals("/")) {
+                        basePath = "";
+                    }
+                    
+                    String fullPath = basePath + path;
+                    String parsedURL = this.parseURL(fullPath, verbs);
+                    element.setPath(parsedURL);
+                    element.setMethod(verb.toString());
+                    element.setFollowRedirects(true);
+
+                    httpSamplers.add(element);
+                }
+            }
+        } catch (NullPointerException e) {
+            // return empty set in case of missing params
+            // TODO log
+            e.printStackTrace();
+            return new HashSet<HTTPSampler>();
+        }
+
+        return httpSamplers;
+    }
+
+    /**
+     * Parses URL and inserts path parameters if exists
+     * @param fullPath
+     * @param verbs    : map containing HttpMethod and Operation pairs
+     * @return a URL containing inserted parameter
+     */
+    public String parseURL(String fullPath, Map<HttpMethod, Operation> verbs) {
+        for (Map.Entry<HttpMethod, Operation> entry : verbs.entrySet()) {
+            List<Parameter> parameters = entry.getValue().getParameters();
+            for (Parameter p : parameters) {
+                if (p.getIn().equals("path")) {
+                    PathParameter pathParam = (PathParameter) p;
+                    if (pathParam.getType().equals("integer")) {
+                        // TODO Full implementation
+                        // currently replaced params with 1
+                        fullPath = fullPath.replace("{" + pathParam.getName() + "}", "1");
+                    }
+                }
+            }
+        }
+
+        return fullPath;
+    }
+
+    /**
+     * Configures a LoopController with the given loop count and httpSampler.
+     * Alternative to createRunTimeController. Returns null if httpSampler is null.
```

Comment:
We don't actually use runtimecontroller for anything anymore. Should probably remove from doc comment